src/Sulu/Bundle/MediaBundle/Entity/BaseCollection.php

```diff
@@ -122,16 +123,6 @@ public function getCreator()
     }
 
     /**
-     * Constructor.
-     */
-    public function __construct()
-    {
-        $this->meta = new \Doctrine\Common\Collections\ArrayCollection();
-        $this->media = new \Doctrine\Common\Collections\ArrayCollection();
-        $this->children = new \Doctrine\Common\Collections\ArrayCollection();
```

Comment:
but these variables are not existing in this class ... they are added in the subclass!